examples/hpc-c/crt0/linker.lds

```diff
@@ -12,17 +12,7 @@ ENTRY(_enter)
 
 MEMORY
 {
-    ram (airwx) : ORIGIN = 0x10000, LENGTH = 0x8000
-}
-
-PHDRS
-{
-    ram PT_LOAD;
-    ram_init PT_LOAD;
-    tls PT_TLS;
-    ram PT_LOAD;
-    text PT_LOAD;
-    lim_init PT_LOAD;
+    ram (airwx) : ORIGIN = 0x100000000, LENGTH = 0x200000000
```

Comment:
The length should be 0x7000_0000 for HPC, since it also includes the `sdram_truncated` section of memory.
```suggestion
    ram (airwx) : ORIGIN = 0x100000000, LENGTH = 0x70000000
```